baseapp/baseapp.go

```diff
@@ -751,6 +756,7 @@ func (app *BaseApp) runMsgs(ctx sdk.Context, msgs []sdk.Msg, mode runTxMode) (*s
 		// Note: Each message result's data must be length-prefixed in order to
 		// separate each result.
 		events = events.AppendEvents(msgEvents)
+		historicalEvents = append(historicalEvents, msgEvents.ToABCIEvents()...)
```

Comment:
does this need to be used somewhere? it looks like its collecting all the msgs events, but its not given to anything else?